daemon/daemon.go

```diff
@@ -255,6 +257,18 @@ func New(conf *Config) (*daemon, error) {
 		CAFile:   conf.Database.CertConfig.CAFile,
 	}
 
+	vpnConfig := wg.WireGuardConfig{
+		Endpoint: conf.VPNConn.Endpoint,
+		Port:     conf.VPNConn.Port,
+		AuthKey:  conf.VPNConn.AuthKey,
+		SignKey:  conf.VPNConn.SignKey,
+		Enabled:  conf.VPNConn.CertConf.Enabled,
+		CertFile: conf.VPNConn.CertConf.CertFile,
+		CertKey:  conf.VPNConn.CertConf.CertKey,
+		CAFile:   conf.VPNConn.CertConf.CAFile,
+		Dir:      conf.VPNConn.Dir,
+	}
+
```

Comment:
service configurations can be divided into another helper  function or somewhere to make functions much more tiny.